Makefile

```diff
@@ -246,3 +247,13 @@ format:
 .PHONY: all build-linux install format lint \
 	go-mod-cache draw-deps clean build build-contract-tests-hooks \
 	test test-all test-build test-cover test-unit test-race benchmark
+	
+	
+###############################################################################
+###                                Tools                                    ###
+###############################################################################
+
+tools:
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.45.0
+	go install mvdan.cc/gofumpt@latest
+
```

Comment:
```suggestion
```

Just add `_ mvdan.cc/gofumpt` to `tools.go`

Reply:
and now I get it......

OK :)